projects/media-viewer/src/lib/annotations/annotations.component.html

```diff
@@ -0,0 +1,8 @@
+<div [ngClass]="'rotation rot' + rotate" [style.width]="width + 'px'" [style.height]="height + 'px'">
+  <ng-container *ngFor="let annotation of annotations" >
+    <mv-annotation [annotation]="annotation"
+                   [zoom]="zoom"
+                   [selectedAnnotation]="selectedAnnotation">
```

Comment:
this is not needed anymore, as we're just listening to the focusOut event on the annotation itself